src/api/v1/users/register/visitors.ts

```diff
@@ -64,7 +65,7 @@ export default [
        * (Can possibly eventually be removed into pre-requisites)
        */
       // Check user doesn't already exist
-      if (await Visitors.exists(knex, { where: { email: payload.email } })) {
+      if (await Users.exists(knex, { where: { email: payload.email } })) {
```

Comment:
Do we want to support visitors signing up who have other types of roles?

Reply:
See https://github.com/TwinePlatform/twine-api/issues/247#issuecomment-443182884

Will add comment to the code